ci/check-update-build-webide.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+GITHUB_API_TOKEN=${GITHUB_API_TOKEN:-""}
+GOOGLE_APPLICATION_CREDENTIALS_CONTENT=${GOOGLE_APPLICATION_CREDENTIALS_CONTENT:-""}
+
+#sets LATEST_SDK_WITH_TIME
+getLatestSdk() {
+  local TMP_FILE=$(mktemp)
+  local JQ_COMMAND='. | map(select(.prerelease == false)) | map([.tag_name,.published_at]) | map(.[0] + "," + .[1] + "," + (.[1]|fromdate|tostring))[]'
+  if [ -n "$GITHUB_API_TOKEN" ]; then
+    $(curl -H "Authorization: $GITHUB_API_TOKEN" https://api.github.com/repos/digital-asset/daml/releases -s > "$TMP_FILE")
+  else
+    $(curl https://api.github.com/repos/digital-asset/daml/releases -s > "$TMP_FILE")
+  fi
+  local RELEASES=$(jq -r "$JQ_COMMAND" "$TMP_FILE")
+  local LAST_COMMAND_RESULT=$?
+  if [ $LAST_COMMAND_RESULT -eq 2 ]; then
+    echo "ERROR: could not read daml releases `cat $TMP_FILE`"
+    return 1
+  fi
+  LATEST_SDK_WITH_TIME=$(echo $RELEASES | awk '{if(substr($1,1,1)=="v") print substr($1,2); else print substr($1,1);}')
+}
+
+dockerAuth() {
+  if [ -n "$GOOGLE_APPLICATION_CREDENTIALS_CONTENT" ]; then
+    GCS_KEY=$(mktemp)
+    echo "$GOOGLE_APPLICATION_CREDENTIALS_CONTENT" > $GCS_KEY
+    gcloud auth activate-service-account --key-file=$GCS_KEY
+
+    docker-credential-gcr config --token-source="gcloud"
+    docker-credential-gcr configure-docker
+  fi
+}
+
+#sets WEBIDE_TS
+getImageTimestampByVersion() {
+  local VERSION=$1
+  local TMP_FILE=$(mktemp)
+  #complications here are parsing the datetime and conforming it to strptime readable dates
+  local JQ_COMMAND=". | map(select(.tags as \$t | \"$VERSION\" | IN(\$t[]))) | map(.timestamp.datetime)[] | sub(\" \";\"T\") | sub(\"(?<before>.*):\"; (.before)) | strptime(\"%Y-%m-%dT%H:%M:%S%z\") | todate | fromdate"
+  #echo "running jq command: $JQ_COMMAND"
+  $(gcloud container images list-tags gcr.io/da-gcp-web-ide-project/daml-webide --format=json > $TMP_FILE)
```

Comment:
I'm not sure I understand the reasoning behind this. What is `gcloud container images list-tags` returning? Why would we ever care about timestamps?

Reply:
I'm checking for timestamps so that if a version of the sdk was updated I will rebuild as well...

Reply:
We never update existing releases.

Reply:
hey - I'm going to grab the timestamps anyway just for logging information. But I'm not basing any logic on the timestamp anymore